sdk/src/main/java/com/horizen/utils/Utils.java

```diff
@@ -13,6 +13,10 @@ private Utils() {}
 
     public static final byte[] ZEROS_HASH = new byte[32];
```

Comment:
is ZEROS_HASH related to a SHA256Hash? If so, instead of `new byte[32];` maybe use `new byte[SHA256_LENGTH];`